src/Client.ts

```diff
@@ -21,7 +21,7 @@ export const AllowedHosts: Host[] = [
  * Represents the main chatexchange Client class.
  * @class
  */
-class Client {
+export class Client {
```

Comment:
Why we exporting a default, AND the class now? If we do this, we should do for all or none. Take your pick.

Reply:
Well, I don't think this is an "all or nothing" choice - since this is a utility package, I think it is important to provide consumers with options on whether they want to import via the default export or use named exports. Besides, with named exports, consumers do not need to do things like `const { default: Client } = await import("...")` when using dynamic imports. But it would not be wise of us to remove the default export too because there already are bots (including the ElectionBot) that use the package, and doing this will force them to make updates whereas it is really unnecessary.